### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by delirious-lettuce**
* `procuced` -> `produced`
  * `containg` -> `containing`
  * `easyiest` -> `easiest`
  * `One` -> `Once`
  * `proccess` -> `process`
  * `googdness` -> `goodness`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: python
 
-# for container-y googdness:
+# for container-y goodness:
```
Comment (cjw296) [inline .travis.yml:3]: Hah, that one probably appears on most of my projects!

File: docs/development.txt [hunk 2]
```diff
@@ -41,7 +41,7 @@ Building the documentation
 --------------------------
 
 The Sphinx documentation is built by doing the following from the
-directory containg setup.py::
+directory containing setup.py::
```
Comment (cjw296) [inline docs/development.txt:44]: Likewise this one...

File: docs/installation.txt [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Installation Instructions
 =========================
 
-The easyiest way to install Mush is::
+The easiest way to install Mush is::
```
Comment (cjw296) [inline docs/installation.txt:4]: ...and this one.

**Review by cjw296 — APPROVED** [review]
Thanks! Much appreciated!

**Comment by delirious-lettuce** [comment]
@cjw296 ,

No problem, thanks for your time!

**Pull request merged**